railties/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Add `unused_routes` command to detect extraneous routes.
+
+    Example:
+
+    ```
+    > bin/rails unused_routes
```

Comment:
I think that this might be better as an option on the existing routes command

```
> bin/rails routes --unused
```

Reply:
I like this, but I also want to keep the command classes separate. I can probably invoke it from the routes command.